src/jshint.js

```diff
@@ -3060,7 +3060,7 @@ var JSHINT = (function() {
       // trigger a warning. Otherwise, the function is safe even within
       // a loop.
       if (f["(isCapturing)"]) {
-        warning("W083", token);
+        warning("W083", token, f["(isCapturing)"].join());
```

Comment:
Mind adding a space to the separator string? As in, ", "

Reply:
Ok.